src/storable.rs

```diff
@@ -492,6 +492,102 @@ where
     };
 }
 
+fn serialize_bounded_with_size<T>(entry: &T, bytes: &mut [u8], start: usize) -> usize
```

Comment:
Docs please (again, related to the point of communication that we discussed). It isn't clear, just reading this function signature what it's meant to return.

Reply:
Done.